imagemagick.yaml

```diff
@@ -34,6 +34,7 @@ environment:
       - perl-dev
       - tiff-dev
       - zlib-dev
+      - openjpeg-dev
```

Comment:
this needs to be formatted with yam

Reply:
Thanks @ajayk for pointing this out, I updated it